pkg/registry/apps/statefulset/storage/storage.go

```diff
@@ -92,3 +115,88 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"sts"}
 }
+
+type ScaleREST struct {
+	registry statefulset.Registry
+}
+
+// ScaleREST implements Patcher
+var _ = rest.Patcher(&ScaleREST{})
+
+// New creates a new Scale object
+func (r *ScaleREST) New() runtime.Object {
+	return &extensions.Scale{}
```

Comment:
> This won't do. We should not just create another "dummy" resource that sits in the extensions group

Oh yeah, totally agree, *but* if you want this to work *now*, that's what you'd have to do.  Otherwise, this actually working with other components will have to wait.